apps/meteor/client/lib/utils/isOverLicenseLimits.ts

```diff
@@ -0,0 +1,22 @@
+import type { LicenseLimitKind } from '@rocket.chat/license';
+
+type Limits = Record<
+	LicenseLimitKind,
+	{
+		max: number;
+		value?: number;
+	}
+>;
+
+export const isOverLicenseLimits = (limits: Limits): boolean => {
+	for (const key in limits) {
+		if (Object.hasOwn(limits, key)) {
+			const limit = limits[key as keyof Limits];
+			if (limit.value !== undefined && limit.value > limit.max) {
+				return true;
+			}
+		}
+	}
+
+	return false;
+};
```

Comment:
`licenses.info` provides a `preventedActions` object, that indicates if a given action is over limits, can we use that?

Reply:
I don't think so... there are some limits that can be of type 'fair usage' this is not prevented in any form